### PR TITLE
That's not how it works

### PR DESCRIPTION
**Pull request by Hendo72**
The `nil` test was done wrong.

TBH, there's no need to test for nil anyway.

if the event isn't "`HEARTHSTONE_BOUND`", then it doesn't matter anyway; return early.

### CODE REVIEW

File: WoWPro/WoWPro_AutoComplete.lua [hunk 1]
```diff
@@ -413,7 +413,7 @@ end
 
 -- Auto-Complete: Set hearth --
 function WoWPro:AutoCompleteSetHearth(event, loc, noUpdate)
-    if event and event ~= "HEARTHSTONE_BOUND" then
+    if event ~= "HEARTHSTONE_BOUND" then
```
Comment (Ludovicus-Maior) [inline WoWPro/WoWPro_AutoComplete.lua:416]: Redundant, but harmless.
